README.md

```diff
@@ -0,0 +1,44 @@
+# Delivery Dashboard
+
+Product Delivery's Web Client to Microservices.
+
+The Product Delivery team is implementing various Microservices such as
+[PollBot](https://github.com/mozilla/pollbot), and this dashboard aims at
+displaying information from those in a centralized place.
+
+## Setting up the development environment
+
+    $ npm i
+    $ ./node_modules/.bin/elm-package install
```

Comment:
Do you think we can add a postinstall script to let `nom i` take care of it? I.e: https://github.com/mozilla/notes/blob/master/package.json#L35